### ISSUE TITLE
permissions profile doesn't work on a non MDS server

### ISSUE THREAD

**Issue by ahusking** [CheckPointSW/CheckPointAnsibleMgmtCollection#83]
Testing on R81.10 management server with ansible [core 2.12.2]

The example to create a user in the ansible docs doesn't actually work.

```
- name: add-administrator
  cp_mgmt_administrator:
    authentication_method: INTERNAL_PASSWORD
    email: admin@gmail.com
    must_change_password: false
    name: admin
    password: secret
    permissions_profile: read write all
    phone_number: 1800-800-800
    state: present
```

it seems to be expecting a dictionary or a list, however when you supply it appropriately it just doesn't seem to get the data and error's out.

Looking at the following code:
https://github.com/CheckPointSW/CheckPointAnsibleMgmtCollection/blob/master/plugins/modules/cp_mgmt_administrator.py

making the below change works for non multi domain servers, but I don't have an environment to test MDS.

add:
       ` permissions_profile=dict(type='str'),`
remove:
 ```
       permissions_profile=dict(type='list', elements='dict', options=dict(
            profile=dict(type='str')
        )), 
```

**Comment by chkp-edenbr** [comment]
Hi @ahusking 

Thanks for updating us we will investigate this.

Regards,
Eden

**Comment by chkp-edenbr** [comment]
Hi @ahusking 

Issue is fixed in new release v4.0.0.

Regards,
Eden
